providers/volatile/api_handler.go

```diff
@@ -35,48 +32,28 @@ var DefaultAPIConfig = config.APIConfig{
 }
 
 // APIHandler implements the APIHandler interface.
-type APIHandler struct {
-	tickers map[mmtypes.Ticker]bool
-}
+type APIHandler struct{}
 
 // NewAPIHandler is invoked by the API factory to create the volatile api handler.
-func NewAPIHandler(market types.ProviderMarketMap) (types.PriceAPIDataHandler, error) {
-	if market.Name != Name {
-		return nil, fmt.Errorf("expected market config name to be %s, got %s", Name, market.Name)
-	}
-	v := APIHandler{
-		tickers: make(map[mmtypes.Ticker]bool),
-	}
-
-	for ticker := range market.TickerConfigs {
-		v.tickers[ticker] = true
-	}
-	return &v, nil
+func NewAPIHandler() types.PriceAPIDataHandler {
+	return &APIHandler{}
 }
 
 // CreateURL is a no-op in volatile api handler.
-func (v *APIHandler) CreateURL(_ []mmtypes.Ticker) (string, error) {
+func (v *APIHandler) CreateURL(_ []types.ProviderTicker) (string, error) {
 	return "volatile-exchange-url", nil
 }
 
 // ParseResponse returns the same volatile price for each ticker.
-func (v *APIHandler) ParseResponse(tickers []mmtypes.Ticker, _ *http.Response) types.PriceResponse {
+func (v *APIHandler) ParseResponse(tickers []types.ProviderTicker, _ *http.Response) types.PriceResponse {
 	var (
 		resolved   = make(types.ResolvedPrices)
 		unresolved = make(types.UnResolvedPrices)
 	)
 
 	volatilePrice := GetVolatilePrice(time.Now, Amplitude, Offset, Frequency)
-
 	for _, ticker := range tickers {
-		if _, ok := v.tickers[ticker]; ok {
-			resolved[ticker] = types.NewPriceResult(volatilePrice, time.Now())
-		} else {
-			err := fmt.Errorf("failed to resolve ticker %s", ticker)
-			unresolved[ticker] = providertypes.UnresolvedResult{
-				ErrorWithCode: providertypes.NewErrorWithCode(err, providertypes.ErrorUnknownPair),
-			}
-		}
+		resolved[ticker] = types.NewPriceResult(volatilePrice, time.Now())
 	}
```

Comment:
This is returning the same price for every ticker, which was the behavior beforehand but im not sure if its intended. gonna leave this as is